WordPress/gutenberg
# Add a vertical toolbar for zoom out mode

## What?


Closes https://github.com/WordPress/gutenberg/issues/59737

Ariakit supports vertical toolbars so we implemented a new variant to the main `Toolbar` component and applied it to a new toolbar specifically for the zoom-out mode.

## Why?


Zoom-out mode needs a toolbar that doesn't cover the content we are inserting. We needed to leverage the Ariakit toolbar for accessibility, so the black pseudo-toolbar that we were using wasn't enough.

## How?


By implementing a toolbar that extends `NavigableToolbar` and adding a new orientation prop that ariakit uses

## Testing Instructions





Open the inserter and open a pattern category
Click on a section, the toolbar should be visible on the side. All buttons should work as usual

### Testing Instructions for Keyboard


When you focus on a section, move focus to the toolbar and use up and down arrows to navigate between the toolbar buttons

## Screenshots or screencast 

<img width="1509" alt="Screenshot 2024-05-10 at 11 38 44" src="https://github.com/WordPress/gutenberg/assets/3593343/6f6149dc-d157-47f7-8167-03a820a6c7a2">
